index.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const readFileHelper = require('./lib/bitmap-file-helper.js');
+
+module.exports = exports = {};
+
+switch (process.argv[2]) {
```

Comment:
@brittdawn @jonathanheemstra @peterkim2 generally speaking, it's bad practice to use `switch` statements but this seems like an interesting use case for one - that said, the following code could be refactored to be more dynamic and with way fewer lines of code - you could just grab the command line argument and pass that as the final parameter of your `readFileHelper`